src/InvariantTest.sol

```diff
@@ -35,6 +47,24 @@ contract InvariantTest is Test {
         excludedSenders_ = _excludedSenders;
     }
 
+    function targetArtifact(string memory newTargetedArtifact_) internal {
+        _targetedArtifacts.push(newTargetedArtifact_);
+    }
```

Comment:
@joshieDo do all of these methods dedupe the arrays in forge or should we check for duplicates when adding to the array?

Reply:
Good question

Reply:
they're deduped yeah.... but to be honest if we're talking about `targetArtifactSelector` they might be getting replaced instead of appended for the same artifact. i'll have to check it out.